web/src/components/apps/Dashboard.jsx

```diff
@@ -498,6 +515,8 @@ class Dashboard extends Component {
                     url={this.props.match.url}
                     isSnapshotAllowed={app.allowSnapshots && isVeleroInstalled}
                     isVeleroInstalled={isVeleroInstalled}
+                    startManualSnapshot={() => this.startManualSnapshot()}
```

Comment:
try to avoid fat arrows when you're not passing any parameters
```suggestion
                    startManualSnapshot={this.startManualSnapshot}
```